src/common/SConscript

```diff
@@ -78,7 +79,7 @@ def scons():
     prereqs.require(env, 'isal_crypto')
 
     denv = env.Clone(LIBS=[])
-    prereqs.require(denv, 'isal', 'isal_crypto', 'protobufc')
+    prereqs.require(denv, 'isal', 'isal_crypto', 'protobufc', 'argobots')
```

Comment:
I would suggest putting this only in daos_common_pmem.so (which is the server side common library).  See the else clause in build_daos_common where pmdk is added to the build.  We could add argobots and only compile this source file in that library.  That may solve some of your issues.

I believe you will also want to add abt to the common_libs.extend line in that function.